playbooks/wordpress.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Deploy the OpenCraft landing page
+  hosts: wordpress
+  become: true
+  vars:
+    - nginx_remove_default_vhost: true
+    - php_memory_ilimit: "128M"
+    - php_max_execution_time: "90"
+    - php_upload_max_filesize: "256M"
+    - php_version: "5.6"
```

Comment:
@Agrendalath, is there any reason for using the `5.6` version which is no longer supported when Ubuntu 16.04 has the 7.0 version?

Reply:
I don't know if the current version of Wordpress and its plugins are fully compatible with it. If we updated it, we would need to check if everything is working correctly there and I don't have enough context on what might break here. So it's possible, but would require more time to test it properly.

Reply:
@Agrendalath, I run Wordpress sites using PHP 7 and they work fine. But it is out of scope for this PR and should be looked at in a different task. We might ditch PHP for the v3 version of the website.